react-native/react/search/index.desktop.js

```diff
@@ -80,20 +80,21 @@ export default class Search extends Component {
   }
 
   render () {
```

Comment:
This isn't a real implementation so let's just comment this out. eventually it'll use material-ui and not this one-off library